app/src/main/res/layout-sw600dp-land/question_player_fragment.xml

```diff
@@ -120,36 +147,6 @@
         tools:layout_constraintGuide_percent="0.5f" />
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-    <FrameLayout
```

Comment:
These changes are not correct as per this comment: https://github.com/oppia/oppia-android/pull/2540#pullrequestreview-578738098

Also you have removed this `hints_and_solution_fragment_placeholder ` which means that `Hints&Solution` won't work in tablet-landscape